modules/Qnite/plugin/qniteaxis.cpp

```diff
@@ -104,6 +121,22 @@ void QniteAxis::setUpperBound(qreal bound)
     qDebug() << "upper changed" << bound;
     m_upperBound = bound;
     emit upperBoundChanged();
+
+    initTicker();
+  }
+}
+
+bool QniteAxis::flip() const
```

Comment:
`flip()` seems an imperative action but it's actually a getter - what about something like `flipState` to avoid confusion?